src/lib/converters/files.converter.ts

```diff
@@ -36,7 +36,9 @@ async function generateTSXFiles(conversionOptions: FilesConversionOptions) {
     iconsFolderName,
     exportCompleteIconSet,
     completeIconSetName,
-    barrelFileName
+    barrelFileName,
+    interfaceName,
+    generateType
```

Comment:
I just saw that the `README` for the type option is wrong. It currently states that the `generateType` option will generate an enum. This is not true. It would be great if we could adjust this as well.